app/models/bookings/phase.rb

```diff
@@ -16,4 +16,12 @@ class Bookings::Phase < ApplicationRecord
 
   acts_as_list
   default_scope -> { order(:position) }
+
+  def self.secondary
+    find_by! name: 'Secondary'.freeze
+  end
+
+  def self.college
+    find_by! name: '16 plus'.freeze
```

Comment:
Is it worth discussing this 'mapping' on Monday and seeing if we can _standardise_ on 'College'? I think it'll simplify things in the long run